client/src/templates/Challenges/classic/classic.css

```diff
@@ -190,12 +190,16 @@
 
 #mobile-layout .portal-button-wrap {
   position: absolute;
-  top: calc(var(--header-height) + var(--breadcrumbs-height));
+  top: auto;
   right: 0;
   height: 2rem;
   border-bottom: 1px solid var(--quaternary-color);
 }
 
+#mobile-layout .tab-content[data-state='active'] .portal-button-wrap {
+  transform: translateY(-2rem);
+}
```

Comment:
the portal button wrap is outside of the tab-content

Reply:
In the mobile layout, the `portal-button-wrap` is outside of the tab-content when the preview tab is not selected.

Although when the preview tab content is displayed, the wrapper which is outside of the tab-content would not be displayed, while the preview tab content includes the wrapper.

So when the keyboard focus is on the Preview tab item, pressing Tab will focus the portal button (which is the first element in the tab panel).

Reply:
I see, the portal button can be removed altogether for the mobile layout in another pr.